stubs/Pillow/PIL/Image.pyi

```diff
@@ -31,53 +34,18 @@ _Color: TypeAlias = Union[int, tuple[int], tuple[int, int, int], tuple[int, int,
 class _Writeable(SupportsWrite[bytes], Protocol):
     def seek(self, __offset: int) -> Any: ...
 
-# obsolete
-NORMAL: Literal[0]
-SEQUENCE: Literal[1]
-CONTAINER: Literal[2]
+NORMAL: Literal[0]  # deprecated
+SEQUENCE: Literal[1]  # deprecated
+CONTAINER: Literal[2]  # deprecated
 
 class DecompressionBombWarning(RuntimeWarning): ...
 class DecompressionBombError(Exception): ...
 
 MAX_IMAGE_PIXELS: int | None
 
-NONE: Literal[0]
```

Comment:
There's 3 sets of changes here:
1. Remove constants that also got removed
2. Add a comment for deprecated constants
3. Some global constants are now generated differently and extracted from enums. I've made that clearer by grouping them with their respective enums and adding a comment. To hopefully help maintainability.